build.gradle

```diff
@@ -29,13 +29,13 @@ repositories {
 }
 
 dependencies {
-	compile 'com.datastax.cassandra:cassandra-driver-core:2.1.7.1'
+	compile 'com.datastax.cassandra:cassandra-driver-core:3.0.1'
 	compile 'com.google.guava:guava:18.0'
 
 	testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
 	testCompile 'org.codehaus.groovy:groovy-all:2.4.4'
 	testRuntime 'io.netty:netty-all:4.1.0.Beta5'
-	testCompile 'org.cassandraunit:cassandra-unit:2.1.9.2'
+	testCompile 'org.cassandraunit:cassandra-unit:3.0.0.1'
```

Comment:
If you leave the cassandra unit to 2.1 we should be fine to upgrade the driver as needed.